frontend/packages/process-editor/src/components/ConfigPanel/ConfigContent/ConfigIcon/ConfigIcon.tsx

```diff
@@ -1,5 +1,6 @@
 import React from 'react';
 import classes from './ConfigIcon.module.css';
+import { ReceiptIcon } from '@navikt/aksel-icons';
 import type { BpmnTaskType } from '../../../../types/BpmnTaskType';
 import { ConfirmationTaskIcon, DataTaskIcon, FeedbackTaskIcon, SignTaskIcon } from '@studio/icons';
```

Comment:
Du kan hente ikonet fra `@studio/icons`.
```suggestion
import type { BpmnTaskType } from '../../../../types/BpmnTaskType';
import { ConfirmationTaskIcon, DataTaskIcon, FeedbackTaskIcon, ReceiptIcon, SignTaskIcon } from '@studio/icons';
```

Reply:
Skal jeg lage en ny fil og kopiere over svgen fra aksel? Ettersom vi ikke har det ikonet der fra før ☺️

Reply:
Alle ikonene fra Aksel skal allerede være inkludert i den pakken. Får du det ikke opp?